common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -436,10 +436,10 @@
   "feature.wynntils.wynncraftButton.name": "Wynncraft Button",
   "feature.wynntils.wynncraftPauseScreen.classSelectionButton.name": "Class Selection",
   "feature.wynntils.wynncraftPauseScreen.hubButton.name": "Back to Hub",
-  "feature.wynntils.wynncraftPauseScreen.name": "Wynncraft Custom Pause Screen",
+  "feature.wynntils.wynncraftPauseScreen.name": "Wynntils Pause Screen",
   "feature.wynntils.wynncraftPauseScreen.wynntilsMenuButton.name": "Wynntils Menu",
-  "feature.wynntils.wynntilsPauseScreen.name": "Wynntils Custom Pause Screen",
-  "feature.wynntils.wynntilsQuestBook.name": "Wynncraft Custom Quest Book",
+  "feature.wynntils.wynntilsPauseScreen.name": "Wynntils Pause Screen",
```

Comment:
This is the same key twice..?

Reply:
wynncraft vs wynntils. I plan to clean up i18n file as there is a few unused keys. Perhaps make a script even.

Reply:
And yes, good idea!